lib/dotcom/system_status/alerts.ex

```diff
@@ -92,7 +88,7 @@ defmodule Dotcom.SystemStatus.Alerts do
   }
   """
   def filter_relevant(alerts) do
-    alerts |> Enum.filter(fn %{effect: effect} -> effect in @service_effects end)
+    alerts |> Enum.filter(&service_impacting_alert?/1)
```

Comment:
Nice!

I could eventually see this whole module getting lumped into `Dotcom.Alerts`, but that's a project for a future day.